frameworks/JavaScript/hapi/app.js

```diff
@@ -8,8 +8,8 @@ var cluster = require('cluster'),
 	Hapi = require('hapi'),
 	mongoose = require('mongoose'),
 	async = require('async'),
-	conn = mongoose.connect('mongodb://localhost/hello_world'),
-	connMap = { user: 'benchmarkdbuser', password: 'benchmarkdbpass', database: 'hello_world', host: 'localhost', charset: 'utf8' };
+	conn = mongoose.connect('mongodb://127.0.0.1/hello_world'),
+	connMap = { user: 'benchmarkdbuser', password: 'benchmarkdbpass', database: 'hello_world', host: '127.0.0.1', charset: 'utf8' };
```

Comment:
Think you accidentally committed the 127.0.0.1 change. Switch back to localhost.